### PR TITLE
removing unwanted wait causing issues for naviagation and asseration

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### PR Objective
Remove this unwanted waiting for the result as in throughout robottelo code uses search entity after removal or deletion. One of the reasons I wanted to remove this was causing a navigation issue.

###
Robottelo PR for test result https://github.com/SatelliteQE/robottelo/pull/7212

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
> test_positive_remove_promoted_cv_version_from_default_env
We had similar situations where we removed the waiting for the element after removal or deletion. e.g. https://github.com/SatelliteQE/airgun/pull/362
 
This was one of the quickest best approaches I understood. If you have any good approach we can work with please let me know.  
  
Concern about tests test_positive_remove_promoted_cv_version_from_default_env ?

```
Testing started at 8:43 PM ...
/home/okhatavk/Satellite/robottelo/env/bin/python /home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_contentview.py::test_positive_remove_promoted_cv_version_from_default_env
Launching py.test with arguments test_contentview.py::test_positive_remove_promoted_cv_version_from_default_env in /home/okhatavk/Satellite/robottelo/tests/foreman/ui

2019-07-22 20:43:30 - conftest - DEBUG - Registering custom pytest_configure

2019-07-22 20:43:30 - conftest - DEBUG - Fetching BZs to deselect...

2019-07-22 20:43:53 - conftest - DEBUG - Deselected tests reason: BZ resolution ['1226425', '1199150', '1479291', '1311113', '1204686', '1475443', '1147100', '1230902', '1217635', '1278917', '1347658', '1487317', '1214312', '1414821', '1310422', '1156555']

2019-07-22 20:43:53 - conftest - DEBUG - Deselected tests reason: missing version flag ['1378442', '1226425', '1610309', '1199150', '1479291', '1701118', '1716429', '1311113', '1581628', '1204686', '1475443', '1147100', '1230902', '1217635', '1278917', '1347658', '1489322', '1625783', '1194476', '1487317', '1682940', '1436209', '1214312', '1310422', '1156555', '1701132']

2019-07-22 20:43:53 - conftest - DEBUG - Deselected tests reason: sat-backlog ['1378442', '1226425', '1610309', '1199150', '1479291', '1701118', '1716429', '1311113', '1581628', '1204686', '1475443', '1147100', '1230902', '1217635', '1278917', '1347658', '1489322', '1625783', '1194476', '1487317', '1682940', '1436209', '1214312', '1310422', '1156555', '1701132']

============================= test session starts ==============================
platform linux -- Python 3.4.8, pytest-4.0.2, py-1.7.0, pluggy-0.9.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-1.0.2, cov-2.6.12019-07-22 20:43:54 - conftest - DEBUG - Collected 1 test cases

collected 1 item
                                                    [100%]

========================== 1 passed in 395.28 seconds ==========================
Process finished with exit code 0
```

File: airgun/entities/contentview.py [hunk 1]
```diff
@@ -239,12 +239,6 @@ def remove_version(self, entity_name, version_name, completely=True,
         view.confirm_remove.click()
         view.flash.assert_no_error()
         view.flash.dismiss()
-        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
-        result = view.versions.search(version_name)
-        if completely and not result:
-            return
```
Comment (mirzal) [inline airgun/entities/contentview.py]: Use the same idiom you use for `Table`s in Robottelo:
```python
if completely and result[0]['Version'] != version_name:
    return
```

Comment (omkarkhatavkar) [reply airgun/entities/contentview.py]: This was working until I was getting the naviagation issue. It was not navigating to next view.

Comment (mirzal) [reply airgun/entities/contentview.py]: On which test? I tried `test_positive_remove_cv_version_from_default_env` and `test_positive_remove_promoted_cv_version_from_default_env` - both passed for me locally, on latest 6.6 snap.

Comment (omkarkhatavkar) [reply airgun/entities/contentview.py]: let me try again.

File: airgun/entities/contentview.py [hunk 2]
```diff
@@ -241,10 +241,8 @@ def remove_version(self, entity_name, version_name, completely=True,
         view.flash.dismiss()
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         result = view.versions.search(version_name)
-        if completely and not result:
-            return
-        view.versions.table.row(
-            version=version_name)['Status'].widget.wait_for_result()
+        if completely and result[0]['Version'] != version_name:
+            return view.versions.table.row(version=version_name)['Status'].widget.wait_for_result()
```
Comment (mirzal) [inline airgun/entities/contentview.py]: Why this change of logic?

Old version was: if `completely == True` and "row is removed from table", then return; otherwise, call `wait_for_result()` on widget.
New version is: if `completely == True` and "row is removed from table", call `wait_for_result()` on widget (widget that is not here anymore); otherwise, do nothin.

**Comment by omkarkhatavkar** [comment]
Let me close this PR and reframe again
